contracts/Vault.vy

```diff
@@ -1472,10 +1470,12 @@ def _creditAvailable(strategy: address) -> uint256:
     # See note on `creditAvailable()`.
     if self.emergencyShutdown:
         return 0
+    if self.debtRatio == 0:
```

Comment:
Could probably do this
```suggestion
    if self.debtRatio == 0 or self.strategies[strategy].debtRatio == 0:
```

But add caching for those two values e.g. `strategy_debtRatio` and `debtRatio` memory variables

Reply:
yes